### ISSUE TITLE
`cluster`: develop downstream usage and visualization

### ISSUE THREAD

**Issue by bluegenes** [sourmash-bio/sourmash_plugin_branchwater#248]
- optionally output dot file of graph?
- tsne/other cluster visualization
- ref @mr-eyes work with kSpider/dbretina

**Comment by ctb** [comment]
yes - it would be great to put together some simple/straightforward viz tutorials!

**Comment by ctb** [comment]
the [betterplot plugin](https://github.com/sourmash-bio/sourmash_plugin_betterplot/) now provides the following commands for working with `pairwise` and `cluster` output:

* `mds2` - MDS plot of `pairwise` output, with optional coloring by categories
* `tsne2` - tSNE plot of `pairwise` output, with optional coloring by categories
* `cluster_to_categories` - build categories files from `cluster` output

See [the Snakefile under `examples/`](https://github.com/sourmash-bio/sourmash_plugin_betterplot/blob/main/examples/Snakefile) in the betterplot github repo for a functioning example!
